projects/planner/src/app/gas-blender/gas-blender.component.spec.ts

```diff
@@ -37,6 +53,20 @@ describe('GasBlenderComponent', () => {
         fixture.detectChanges();
         const calc = TestBed.inject(GasBlenderService);
         calculateSpy = spyOn(calc, 'calculate');
+        simplePage = new GasBlenderPage(fixture);
+    });
+
+    it('should toggle pricing and display correct total price', () => {
+
+        simplePage.pricingToggleBtn.click();
+        fixture.detectChanges();
+        expect(component.showPricing).toBeTruthy();
```

Comment:
Is part of the other test, use UI to detect, if the element is shown on the page instead.